pages/implementation/reference/javascript.mdx

```diff
@@ -0,0 +1,190 @@
+
+> **Arguments**
+>
+> `options`: `{env, [noop], [strict], [avoLogger], [inspector], [mobileDebugger]}`
+>   - `env`: string, one of 'dev', 'prod' or 'staging'. 
+>  - `[noop = false]`: optional bool, if set, Avo won't make any network calls (no tracking) in development and staging environments. Note that the noop flag is ignored in production. 
+>   - `[strict = false]`: optional bool, if set, Avo will throw an exception when it detects a tracking problem in development or staging. Note that the noop flag is ignored in production. 
+>   - `[avoLogger]`: optional custom implementation of the logger. Can be used to disable logs. <Link passHref href="/implementation/custom-loggers"><a>More about custom loggers</a></Link>  
+>   - `[inspector]`: optional Avo Inspector object. If you use Avo Inspector pass it here to make the Avo functions automatically report the invocations to the Avo Inspector. 
+>   - `[mobileDebugger]`: React Native specific and optional Avo Debugger instance. Pass it to make the Avo functions automatically show the functions calls and all the errors in the visual debugger. Check <a target="\_blank" rel="noopener noreferrer" href="https://github.com/avohq/react-native-analytics-debugger">React Native mobile debugger repo</a> to learn more about it.
+>
+> `systemProperties`: an object where each field represents a system propety in your tracking plan. 
+> When you define system properties in your Avo workspace you set name and type - the fileds of this object are named the same as system propertyies, in lower camel case, and you should provide corresponding types.
+>
+> `destinationOptions`: `{[segment], [amplitude], [mixpanel]}`
+>   - `mixpanel`: optional object, if you use Mixpanel destination managed by Avo, this object will be passed to `mixpanel.init(apiKey, options)` as the second parameter, `options`
+>   - `amplitude`: optional object, if you use Amplitude destination managed by Avo, this object will be passed to `amplitude.init(apiKey, null, options)` as the third parameter, `options`
+>   - `segment`: optional object, if you use Segment destination managed by Avo, this object will be passed to `analytics.load(apiKey, options)` as the second parameter, `options`
+>
+> `customDestination`: required if given source has a custom destination set up in Avo. Otherwise ignored.
+>
+> Read more about custom destinations <Link passHref href="/implementation/strat-custom-destination"><a>here</a></Link>.
+>
+> Find examples of JavaScript custom destinations for each analytics destination in <Link passHref href="/implementation/destinations"><a>this section</a></Link>.
+
+##### setAvoLogger
+
+```javascript
+Avo.setAvoLogger(avoLogger)
+```
+
+Separarte method to set custom logger
+
+> **Arguments**
+>
+> `avoLogger`: custom implementation of the logger. Can be used to disable logs. <Link passHref href="/implementation/custom-loggers"><a>More about custom loggers</a></Link>  
```

Comment:
Moved this description above. Here we should have a description of the `avoLogger` I need to pass in. What is it?